Sources/ASTPreprocessor/ASTPreprocessor.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ASTPreprocessor.swift
+//  ASTPreprocessor
+//
+//  Created by Nik Vangerow on 11/22/18.
+//
+
+import AST
+
+/// The Preprocessing pass for the AST.
```

Comment:
More descriptive? E.g. `performs tree transformations`?

Reply:
Perhaps. The others are also not very descriptive but I suppose I can do this